Source/WebCore/PAL/pal/spi/cocoa/QuartzCoreSPI.h

```diff
@@ -153,6 +157,10 @@ typedef struct _CARenderContext CARenderContext;
 @property (getter=isSeparated) BOOL separated;
 #endif
 @property BOOL toneMapToStandardDynamicRange;
+#if PLATFORM(VISION)
+@property (nonatomic) CGFloat sizeMultiplier;
+@property (nonatomic, copy) void (^effectGroupConfigurator)(CARemoteEffectGroup *group);
+#endif
```

Comment:
We don't need this since it's being declared in the implementation file, and doesn't belong here anyway (these are in a RealitySystemSupport header).